client/state/sites/plans/assembler.js

```diff
@@ -10,6 +10,7 @@ const createSitePlanObject = ( plan ) => {
 
 	return {
 		autoRenew: Boolean( plan.auto_renew ), // Always true for plans paid with credits.
+		autoRenewDateMoment: plan.auto_renew_date ? moment( plan.auto_renew_date ).startOf( 'day' ) : null,
```

Comment:
@mtias I think the approach of what we've been doing with computed attributes decorating raw data is clean, but may cause performance issues if the selector is called a lot. 

Should we instead be looking into storing the computed data, with a custom serializer/deserializer to deal with the class instances?

@artpi This particular sub-tree is rather odd, so let's hold off until we have a PR to clean up all of these moment instances.  (Though we should be avoiding adding class instances to new trees without good reason).